subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -52,7 +52,7 @@ static int prepare_cb_common(struct lll_prepare_param *p, uint8_t chan_idx);
 static int is_abort_cb(void *next, void *curr, lll_prepare_cb_t *resume_cb);
 static void abort_cb(struct lll_prepare_param *prepare_param, void *param);
 static int isr_rx(struct lll_sync *lll, uint8_t node_type, uint8_t crc_ok, uint8_t rssi_ready,
-		  enum sync_status status);
+		  enum sync_status status, uint8_t cte_ready);
```

Comment:
Again my preference is to have the `cte_ready` before `status` or before `rssi_ready`, as it is radio related.

Reply:
I've moved `cte_ready` before `rssi_ready`.